lib/puppet_x/puppetlabs/registry.rb

```diff
@@ -6,6 +6,44 @@ module Registry
   # For 64-bit OS, use 32-bit view. Ignored on 32-bit OS
   KEY_WOW64_32KEY = 0x200 unless defined? KEY_WOW64_32KEY
 
+  # Helper methods used in the registry providers
+  # Guarded to only load on Windows due to Win32 constants
+  if Puppet.features.microsoft_windows?
```

Comment:
I don't think I understand why the guard is needed given the constants wouldn't be used unless this code is actually called?

Reply:
The spec tests load the provider code which loads this.

Reply:
Hmm... will have to look closer.  Going to merge for now.